### ISSUE TITLE
docs: Text within Benchmark results page isn't visible properly

### ISSUE THREAD

**Issue by checkaayush** [DiceDB/dice#1364]
![Screenshot 2024-12-07 at 11 20 23 PM](https://github.com/user-attachments/assets/e009e45b-ba4b-4d82-808a-a84fce6ca615)

Page: https://dicedb.io/benchmarks/

**Comment by ayushsatyam146** [comment]
Are you using some dark mode extension? It is perfectly visible to me
<img width="1195" alt="image" src="https://github.com/user-attachments/assets/9fa50ba6-0609-4055-96ca-2555930caee7" />

**Comment by checkaayush** [comment]
Nope, I'm not using any dark mode extensions. I'm using latest versions of Chrome (Version 131.0.6778.140 (Official Build) (arm64)) and Safari, and able to reproduce it consistently, including on mobile (iOS).

**Comment by RamGoel** [comment]
Works fine for me too. Maybe some extension conflicts with your browser @checkaayush 

Try once in incongnito

**Comment by checkaayush** [comment]
Same behaviour for me in incognito tab as well. Tried in another browser with no extensions as well (Safari) and have the same issue.

**Comment by RamGoel** [comment]
quite strange, let's wait till someone else also face this?
